release-team/role-handbooks/communications/README.md

```diff
@@ -43,11 +44,59 @@ The list has to be rotated/actively managed every cycle. Submit a PR to update [
+
+As feature blogs are opted in, assign them to shadows and yourself for tracking and facilitation. The responsibility is to ensure the blog authors have the resources and information they need, including editorial and tech reviews once ready.
+
+Work closely with the SIG Docs Blogs team (connecting on `#sig-docs-blogs` and by attending meetings), as they are typically available for editorial reviews. Share with them the feature blog schedule and updates throughout the cycle.
+
+For tech reviews, reach out to authors and the sponsoring SIG to organize at least one tech review per blog post.
+
+### Mid-cycle deprecations and removals blog
+
+This blog is optional and will vary from release to release. Work with the rest of the release team ahead of the Code Freeze date to determine if a mid-cycle blog focused on feature deprecations and removals is warranted. If so, facilitate its creation and publication.
+
+If the release will deprecate important and commonly-used features (or simply a large number of features will be deprecated), consider publishing this blog. Also consider this when commonly-used features that have been deprecated are removed in a release. ['Kubernetes API and Feature Removals in 1.22'](https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/) and ['Deprecated APIs Removed in 1.16'](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) are great examples to work from.
+
+Regardless of reason, the timing for publication is ahead of the release, in order to inform the community and allow for preparation time. Start the discussion mid-cycle and well ahead of Code Freeze, and target publication for Code Freeze week.
+
+### Press and release webinar
+
+This is a light but very important component of the Communications Coordinator role. Two sets of activities need to be scheduled with the CNCF, namely press release and interview scheduling around the release day, and the release webinar after the release.
+
+To schedule the press briefings, you will be a liaison between the Release lead and the CNCF contacts. Send an email about a month ahead of the release and coordinate between the parties to get release day press events scheduled.
+
+To schedule the release webinar with the CNCF, use the Calendly link (below) to schedule a "live webinar". This will typically occur 3-4 weeks after the release, and is primarily presented by the Release lead and Enhancements lead. Often the Comms lead will also join the webinar. The format is open, but primarily the team walks through the enhancements in the release and gives a sneak peek of what's coming in the next release.
```

Comment:
In the past, I've just reached out to CNCF PR directly about this. Live webinars are pretty competitive to sign-up for, especially depending on the timing since they open up quarterly to projects and CNCF member companies. I'd also recommend reaching out maybe ~2 months beforehand (instead of 3-4 weeks) for the same timing issues. Thoughts?

Reply:
I'm cool with that. I've added emails for both pr and webinars. I think the timing is in the right place (4 weeks ahead of release to schedule a webinar 2-4 weeks after). Things look ok?